src/sp_repo_review/checks/pyproject.py

```diff
@@ -94,6 +94,24 @@ def check(pyproject: dict[str, Any], package: Traversable) -> bool | None:
         return None
 
 
+class PP005(PyProject):
+    "Does not have deprecated trove classifiers"
+
+    url = mk_url("packaging-simple")
+
+    @staticmethod
+    def check(pyproject: dict[str, Any], package: Traversable) -> bool | None:
+        """
+        As of Pypi metadata 2.4, all the `License ::` classifiers are deprecated.
+        Prefer the `License-Expression`
+
+        See https://packaging.python.org/en/latest/specifications/core-metadata/#license-expression
+        """
+        match pyproject:
+            case {"project": {"classifiers": classifiers}}:
+                return len([c for c in classifiers if c.startswith("License ::")]) == 0
```

Comment:
This should be guarded by a check for `project.license` being a string. If it's a string, this check is valid. If not, then METADATA<2.4 is probably being used.

We could possibly even do this by backend, we compute that too.